gatsby-config.js

```diff
@@ -82,16 +83,17 @@ module.exports = {
           "default-src": "'self' https://www.google-analytics.com data:",
           "script-src":
             "'unsafe-eval' 'unsafe-inline' 'self' *.earthdata.nasa.gov https://www.googletagmanager.com https://www.google-analytics.com https://fbm.earthdata.nasa.gov https://www.google.com/recaptcha https://www.gstatic.com/recaptcha https://fonts.gstatic.com",
-          "style-src": "'unsafe-inline' 'self' data: *.earthdata.nasa.gov",
+          "style-src":
+            "'unsafe-inline' 'self' data: *.earthdata.nasa.gov https://cdn.knightlab.com",
```

Comment:
I've tested it locally, and it seems we don't need to give permissions to the knightlab.com urls anymore, as we are not loading data from the google spreadsheets.